modulemd/tests/test-modulemd-modulestream.c

```diff
@@ -1021,6 +1021,21 @@ module_stream_test_v2_yaml (void)
   g_clear_pointer (&specV2Path, g_free);
 }
 
+
+static void
+module_packager_v2_sanity (void)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  g_autoptr (GError) error = NULL;
+
+  g_autofree gchar *specV2Path = g_strdup_printf (
+    "%s/yaml_specs/modulemd_stream_v2.yaml", g_getenv ("MESON_SOURCE_ROOT"));
+  stream =
+    modulemd_module_stream_read_file (specV2Path, TRUE, NULL, NULL, &error);
+  g_assert_no_error (error);
+  g_assert_nonnull (stream);
+}
```

Comment:
`g_clear_object (&stream); g_clear_pointer (&specV2Path, g_free);` for good measure?

Reply:
Probably good hygiene.